client/store/orders/selectors.js

```diff
@@ -16,7 +16,7 @@ import { ERROR } from 'store/constants';
  * Returns orders for a specific query.
  *
  * @param  {Object} state     Current state
- * @param  {Object} query     Report query paremters
+ * @param  {Object} query     Report query paremeters
```

Comment:
parameters

Reply:
Ouch. :upside_down_face: 

Should be fixed everywhere, now. 614d905fa0921845362b4ad0987188e834253713